.envrc

```diff
@@ -11,6 +11,10 @@ dotenv_if_exists
 mkdir -p "$PWD/.devenv"
 DEVENV_ROOT_FILE="$PWD/.devenv/root"
 printf %s "$PWD" > "$DEVENV_ROOT_FILE"
+
+git update-index --assume-unchanged nix/test-environments/test-keys/[A-Z]*
+git update-index --skip-worktree nix/test-environments/test-keys/[A-Z]*
```

Comment:
Would benefit from a comment explaining the need and purpose for those two commands, given their peculiarity

Reply:
Might also want to define a more generic way to have some files *ignored* by `git`, at least in most of the meaningful ways